src/support/hazard.c

```diff
@@ -19,7 +19,7 @@ static void __hazard_dump(WT_SESSION_IMPL *);
 static int
 hazard_grow(WT_SESSION_IMPL *session)
 {
-    WT_HAZARD *nhazard;
```

Comment:
This local variable (new hazard array pointer) conflicts with the name of the session field (number of active hazard pointers). Renaming to make it clearer